src/DataLoaders.py

```diff
@@ -0,0 +1,102 @@
+# Classes for wrapping data into datasets
+
+import pandas as pd
+import numpy as np
+import biotite
+import biotite.structure as struc
+import src.DataProcessing  as dp
+from tqdm import tqdm
+from typing import Optional, Literal
+from torch_geometric.data import Dataset
+
+class ProteinDataset(Dataset):
+    """
+    Dataset for loading proteins from the paper given splits.
+    """
+    def __init__(self, labels, database, include, flex: Literal["msqf", "bfact", "pseudo"] = "msqf", exclude: Optional[str] = None):
+        self.flex = flex
+        self.database = database
+        self.structures = []
+        self.protein_labels = pd.DataFrame(columns = ['chain_id', 'label'])
+        if exclude != None:
+            structure_set = set(line.strip() for line in open(include)).difference(set(line.strip().replace("_",".") for line in open(exclude)))
+        else:
+            structure_set = set(line.strip() for line in open(include))
+        for id in tqdm(structure_set):
+            PDBid, chain = id.split(".")
```

Comment:
The normal python convention is to use `CamelCase` for class names, `lower_case_with_underscores` for all other variables and `UPPER_CASE_WITH_UNDERSCORES` for global constants.

I recommend renaming `PDBid` -> `pdb_id`